pkg/model/model.go

```diff
@@ -664,13 +664,23 @@ func (m *Model) ApplyShapeIgnoreRules() {
 		return
 	}
 	for sdkShapeID, shape := range m.SDKAPI.API.Shapes {
+	ignoreFields:
```

Comment:
I don't like this, I can extract this function later... :palm_tree:

Reply:
@chlunde checkout this very similar function :) https://github.com/aws-controllers-k8s/code-generator/pull/78/files#diff-f5c09cb6b638098a0e513f710e901c9f7381165e4e2c49f2b3881613d2db518dR324

Reply:
@chlunde ping! :) Did you have a chance to check out the code and comment from @A-Hilaly above?

Reply:
back from vacation today! I'll try to fix it by tomorrow